### DISCUSSION TITLE
System Info reports a version that doesn't exist

### DISCUSSION THREAD

**Discussion by aidenmitchell** [requarks/wiki#4792] [Error / Bug Report] [answered]
![image](https://user-images.githubusercontent.com/30846409/146259951-972abaa3-a593-4a81-b34c-26959e7eb17d.png)
It wants to upgrade backwards...

**Comment by NGPixel** [comment, marked as answer]
Both versions are identical, you can ignore (or upgrade to the latest 2.5.260)
